### DISCUSSION TITLE
Scramble trying to reach to api.json from localhost, even if it is deployed to production server

### DISCUSSION THREAD

**Discussion by Bekzod101001** [dedoc/scramble#325] [Q&A]
<img width="450" alt="image" src="https://github.com/dedoc/scramble/assets/29505789/b00ceedf-d930-46ff-b597-21e38b9ec216">

i defined another domain in APP_URL like this, but it doesn't working.

also tried to directly set api_domain property in scramble.php config file. But it's still failing.

i tried to directly open api.json from browser, and it is generated. However from /docs/api it's still trying to open an api.json from 127.0.0.1

and BTW, i have defined in Scramble::routes in AppServiceProvider, i was thinking maybe it's overriding default behavior, but it looks like not.

**Comment by romalytvynenko** [comment]
Hey @Bekzod101001

Can you create reproduction repo? Otherwise I won't be able to help.
